### ISSUE TITLE
[Feature Request] PickVideoAsync: Add option to not copy the video

### ISSUE THREAD

**Issue by prawira76** [jamesmontemagno/MediaPlugin#190]
## Feature Request:
For performance and storage reasons, I use PickPhotoAsync() with PhotoSize.Small and a CompressionQuality of 90 for displaying images in a grid. When the image is tapped, the original image is shown in the default photo app using MediaFile.AlbumPath.

I'd like to achieve something similar with PickVideoAsync(). I'm already creating a small video thumbnail, but I'd like to have the option to not copy the video to a directory of my own app. The video can then be played in the default videoplayer using MediaFile.AlbumPath as well.

What do you think?

**Comment by marza91** [comment]
I just tried to take a look at this, as it gives us problems with the whole app freezing while the file is being copied, no possibility to show a loading indicator or similar.
As a first test I just commented out the "StartsWith" check of this line:
https://github.com/jamesmontemagno/MediaPlugin/blob/caef079c36bb5f7497b73820a09b3098ba326c4b/src/Media.Plugin/Android/MediaPickerActivity.cs#L533

Which seems to work for me, the Path variable points directly to the selected file and nothing is copied. Not sure why this check is there, as the path I get in the check is something like "/storage/emulated/0/DCIM/Camera/VID_20180624_120514.mp4" which is then passed on in the MediaFile.Path info. Only tested in Android 8 though, so it might differ between different versions?

I'm also curious about why the files are being copied over in the first place, as reading them from the original location seems just fine to me?
